Sentinel/Classes/Core/PerformanceInfo/PerformanceTool.swift

```diff
@@ -26,18 +25,11 @@ public class PerformanceTool: Tool {
         let toolTable = createToolTable()
         toolTable.presentPreview(from: viewController)
     }
-
-    public func createViewController(on viewController: UIViewController?) -> UIViewController {
-        let toolTable = createToolTable()
-        let controller = toolTable.createViewController()
-        controller.tabBarItem = UITabBarItem(title: "Performance", image: UIImage.Sentinel.performance.resize(), tag: 1)
-        return controller
-    }
 }
 
-// MARK: - Private methods
+// MARK: - Internal methods
 
-private extension PerformanceTool {
+internal extension PerformanceTool {
```

Comment:
```suggestion
extension PerformanceTool {
```
internal is by default, you can check your code for it 😄 

Also please check, I've noticed some comments that stated Private var / methods when in fact internals were used.